llvm/lib/Target/PowerPC/PPCSubtarget.h

```diff
@@ -216,8 +216,8 @@ class PPCSubtarget : public PPCGenSubtargetInfo {
   bool isSVR4ABI() const { return !isAIXABI(); }
   bool isELFv2ABI() const;
 
-  bool is64BitELFABI() const { return  isSVR4ABI() && isPPC64(); }
-  bool is32BitELFABI() const { return  isSVR4ABI() && !isPPC64(); }
+  bool is64BitELFABI() const { return isSVR4ABI() && isPPC64(); }
```

Comment:
Unrelated changes?

Reply:
Yes this is.  It was auto updated by clang-format.  Not sure why it was identified as needing change but I left it since its a valid format update.